Apoorv38.md

```diff
@@ -0,0 +1,52 @@
+I am excited to build this one. Never made something like this.
+I think that everyone would like to have a freindly companion who would accompany you where ever you go and doesn't require much maintanence.
+I'm not sure if it would be as good as I think but it's worth trying.
+
+## Plan
+
+In each leg I will use 3 servo-motors one for controlling the direction of   the pupp, the other two are for forward motion.
+I will use IR proximity sensor and Ultrasonic distance finder to avoid   obstalcesfrom a safe distance.
+I will use esp 32 cam for face recognition of the owner.
+I'll use OLED screen for its face.
+It would be powered using a Lithium Polymer battery(800 mAh).
+I am using two Arduino boards because I think 1 wouldn't be enough for the pupp.
+It would also require other components for connection(e.g. jumper wires, voltage regulators, etc.).
+It will also require 3D printed parts(e.g. the gears, its legs, etc.).
+I also need some tools for the asembling procces.
+
+## Budget
+
+| Product     | Supplier/Link                       | Cost       |
+|-------------|-------------------------------------|------------|
+|Servo motor x12|https://robu.in/product/towerpro-sg90-continuous-rotation-360-degree-servo-motor/| $25.50 |
+|Arduino Nano 33 ble sense|https://robu.in/product/arduino-nano-33-ble-sense-with-header/| $46 |
+|Arduino Micro|https://robu.in/product/original-arduino-micro-no-headers/| $20 |
+|IR proximity sensor x4| https://robu.in/product/waveshare-infrared-proximity-sensor-obstacle-avoiding/| $15 |
+|Esp 32 cam|https://robu.in/product/ai-thinker-esp32-cam-development-board-wifibluetooth-with-ov2640-camera-module/| $11|
+|Micro Breadboard x2|https://robu.in/product/400-tie-points-contacts-mini-circuit-experiment-solderless-breadboard/| $1.50 |
+|OLED 0.96"|https://robu.in/product/0-96-inch-yellow-yellow-blue-oled-lcd-led-display-module/     | $3 |  
+|LiPo 2s 800 mAh|https://robu.in/product/ta-45c-800-2s1p-jst/?pid=1059839| $12.50 | 
+|LiPo balance charger|https://robu.in/product/b3ac-compact-balance-charger-2s-3s-lipo/| $4.50 |
+|3v LDO voltage regulator x4|https://robu.in/product/lds3985m33r-3-3v-300ma-ldo-bicmos-voltage-regulator-5-pin-sot-23-stmicroelectronics/| $4 |
+|Multimeter|https://robu.in/product/mastech-mas830l-digital-multimeter-with-probes-original/| $11|
+|Poteniometer x4|https://robu.in/product/3310y-001-503l-rotary-potentiometer/| $12.50 |
+|Ultrasonic range finder HC-SR04|https://robu.in/product/hc-sr04-ultrasonic-range-finder/| $1 |
+|Logic level convertor|https://robu.in/product/8-channel-bi-directional-logic-level-converter-module/| $1 |
+|Jumper wires male to male x2|https://robu.in/product/male-to-male-jumper-wires-40pcs-20cm/| $1 |
+|Pin male header strip|https://robu.in/product/2-54mm-1x40-pin-male-single-row-straight-short-header-strip-pack-of-3/?pid=1114172| $0.50 |
+|Glue Gun|https://amzn.eu/d/6Jnao54| $5 |
+|Drill|https://amzn.eu/d/7eHzw34| $18 |
```

Comment:
What's the drill for?

Reply:
I need it to make holes in my 3D printed parts.(It is not that necessary but its better to have one)